properties/P000102.md

```diff
@@ -2,8 +2,20 @@
 uid: P000102
 name: Semimetrizable
 refs:
-- mr: 2492954
-  name: ! 'The compact-open topology: A new perspective'
+- zb: "0116.14303"
+  name: A regular semi-metric space for which there is no semimetric under which all spheres are open. (Heath)
 ---
 
-Defined in Definition 3.7 of {{mr:2492954}}
+A space $X$ for which
+there is a distance function (or "semi-metric") $d:X^2\to[0,\infty)$
+satisfying reflexivity ($d(x,y)=0\Leftrightarrow x=y$) and symmetry
+($d(x,y)=d(y,x)$), such that if $p$ is
+a limit point of some $Y\subseteq X$, then $\inf\{d(y,p):y\in Y\}=0$.
+
+Defined in e.g. {{zb:0116.14303}}, where the following equivalence was
+given: the space is {P28} witnessed by $\{G_n(x):n<\omega\}$
```

Comment:
```suggestion
given: if the space is {P28} witnessed by $\{G_n(x):n<\omega\}$
```
Apparently a function such as $G$ here is referred to as a $g$-function, but I'm not sure how much this is worth mentioning.